wkhtmltopdf/utils.py

```diff
@@ -8,7 +8,7 @@
 import shlex
 from tempfile import NamedTemporaryFile
 
-from django.utils.encoding import smart_text
+from django.utils.encoding import force_str
```

Comment:
I think you should use `smart_str` instead.  `smart_text` is an alias of `smart_str`, not `force_str`.

Reply:
Updated. Thanks for catching.